src/policy/feerate.cpp

```diff
@@ -42,3 +42,7 @@ std::string CFeeRate::ToString(const FeeEstimateMode& fee_estimate_mode) const
     default:                      return strprintf("%d.%08d %s/kvB", nSatoshisPerK / COIN, nSatoshisPerK % COIN, CURRENCY_UNIT);
     }
 }
+
+std::string CFeeRate::SatsToString() const {
```

Comment:
style nit d6643ef609bf442fe38dd8a1335dd7c580f1fdd7:

could clang-format?

Reply:
no longer adding this `SatsToString()` function in favor of making the existing `ToString()` more flexible